app/src/main/java/org/mozilla/focus/browser/MainActivity.kt

```diff
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package org.mozilla.focus.activity
+package org.mozilla.focus.browser
```

Comment:
I think MainActivity and ScreenController should be at the top-level (`org.mozilla.focus`): they switch between all the other features, which are composed into different packages, such as `home/` for the home screen feature.